#! /bin/sh /usr/share/dpatch/dpatch-run
## no_sidux-xdg.dpatch by Stefan Lippers-Hollmann <s.l-h@gmx.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: we no longer provide sidux-xdg

@DPATCH@
diff -urNad trunk~/menu/sidux-manual.desktop trunk/menu/sidux-manual.desktop
--- trunk~/menu/sidux-manual.desktop	2007-12-06 18:07:42.000000000 +0100
+++ trunk/menu/sidux-manual.desktop	2008-01-08 20:03:56.000000000 +0100
@@ -5,4 +5,4 @@
 Icon=sidux-book2b
 Name=sidux manual
 Name[de]=sidux manual
-Categories=sidux
+Categories=System;Documentation;
